### PR TITLE
Fix compilation debug profile on windows

### PR DESCRIPTION
**Pull request by sgaragagghu**
Can`t compile with "cargo build" and "cargo test" (so without --release), on windows.

### CODE REVIEW

File: pow/randomx/sys/build.rs [hunk 1]
```diff
@@ -17,7 +17,11 @@ fn main() {
 	let dst = config.build();
 
 	if target.contains("pc-windows-msvc") {
-		println!("cargo:rustc-link-search=native={}/build/Release", dst.display());
+		if env::var("PROFILE").unwrap() == "debug" {
```
Comment (sorpaas) [inline pow/randomx/sys/build.rs:20]:
```suggestion
		if env::var("PROFILE").expect("PROFILE env not found") == "debug" {
```

File: pow/randomx/sys/build.rs [hunk 2]
```diff
@@ -17,7 +17,11 @@ fn main() {
...
+			println!("cargo:rustc-link-search=native={}/build/Debug", dst.display());
```
Comment (sorpaas) [inline pow/randomx/sys/build.rs:21]:
```suggestion
			println!("cargo:rustc-link-search=native={}/build/Debug", dst);
```

File: pow/randomx/sys/build.rs [hunk 3]
```diff
@@ -17,7 +17,11 @@ fn main() {
...
+		} else {
+			println!("cargo:rustc-link-search=native={}/build/Release", dst.display());
```
Comment (sorpaas) [inline pow/randomx/sys/build.rs:23]:
```suggestion
			println!("cargo:rustc-link-search=native={}/build/Release", dst);
```

File: pow/randomx/sys/build.rs (hunk 1 above)
Comment (sgaragagghu) [reply pow/randomx/sys/build.rs]: Right, have been deviated by reading this file and cmake crate where it`s only used unwrap 😞

File: pow/randomx/sys/build.rs (hunk 2 above)
Comment (sgaragagghu) [reply pow/randomx/sys/build.rs:21]: I guess that`s just how it is supposed to be used:
`A Path might contain non-Unicode data. This struct implements the Display trait in a way that mitigates that. It is created by the display method on Path.`
https://doc.rust-lang.org/std/path/struct.Display.html

Comment (sorpaas) [reply pow/randomx/sys/build.rs:21]: Ah you're right!

**Pull request merged**